emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBClient.java

```diff
@@ -476,7 +475,8 @@ protected AwsCredentialsProvider getAwsCredentialsProvider(Configuration conf) {
     }
 
     if (Strings.isNullOrEmpty(accessKey) || Strings.isNullOrEmpty(secretKey)) {
-      providersList.add(InstanceProfileCredentialsProvider.create());
+      log.debug("Custom credential provider not found, loading default provider from sdk");
+      providersList.add(DefaultCredentialsProvider.create());
```

Comment:
Let's add a test in DynamoDBClientTest for checking this

Reply:
good idea ill play around and make a test